aep/general/0135/aep.md.j2

```diff
@@ -71,24 +93,40 @@ message DeleteBookRequest {
 ```
 
 - A `name` field **must** be included. It **should** be called `name`.
-  - The field **should** be [annotated as required][aip-203].
-  - The field **must** identify the [resource type][aip-123] that it
+  - The field **should** be [annotated as required][aep-203].
+  - The field **must** identify the [resource type][aep-123] that it
     references.
 - The comment for the field **should** document the resource pattern.
 - The request message **must not** contain any other required fields, and
   **should not** contain other optional fields except those described in this
-  or another AIP.
+  or another AEP.
+
+{% endtabs %}
 
 ### Soft delete
 
 **Note:** This material was moved into its own document to provide a more
-comprehensive treatment: AIP-164.
+comprehensive treatment: [AEP-164][aep-164].
 
 ### Long-running delete
 
-Some resources take longer to delete a resource than is reasonable for a
+Some resources take longer to delete than is reasonable for a
 regular API request. In this situation, the API **should** use a long-running
-operation instead:
+operation instead. See [AEP-151][aep-151] for full details on long-running operations.
+
+{% tab oas -%}
+
+{% sample 'long_running_delete.oas.yaml', 'paths' %}
+
+- The response status code should be `202 Accepted` if the request was
```

Comment:
Should this guidance be in the LRO AEP rather than here?  (putting aside questions about consistency with gRPC/JSON transcoding)

Reply:
Maybe but this corresponds to the proto guidance below which comes straight from the Google AIP. Let's leave it for now in the interest of getting this merged and we can revisit in a future PR.